src/theory/quantifiers/extended_rewrite.h

```diff
@@ -200,11 +200,17 @@ class ExtendedRewriter
   /** Partial substitute
    *
    * Applies the substitution specified by assign to n, recursing only beneath
-   * terms whose Kind appears in rec_kinds.
+   * terms whose Kind appears in rkinds (when rkinds is empty), and additionally
+   * never recursing beneath WITNESS.
    */
   Node partialSubstitute(Node n,
                          std::map<Node, Node>& assign,
                          std::map<Kind, bool>& rkinds);
+  /** same as above, with vectors */
+  Node partialSubstitute(Node n,
+                         std::vector<Node>& vars,
```

Comment:
As both `vars` and `subs` are not modified, I'd pass them as `const &`. The same is true for `assign` in the previous overload of `partialSubstitute`.

Reply:
Agreed, I'll update.